.github/workflows/yamllint/.yamllint.yml

```
@@ -0,0 +1,20 @@
+extends: default
+
+ignore: |
+    /themes/classic/_dev/node_modules/
+    /themes/node_modules/
+
+rules:
+    braces:
+        min-spaces-inside: 1
+        max-spaces-inside: 1
+    brackets:
+        min-spaces-inside: 1
+        max-spaces-inside: 1
+    document-start: disable
+    indentation:
+        ## Check .editorconfig
+        spaces: 4
```

```suggestion
        spaces: 2
```

Prefer 2as we do for html and json for the moment :D

---

You can read more here https://docs.saltproject.io/en/3000/topics/troubleshooting/yaml_idiosyncrasies.html#:%7E:text=Indentation,uniform%20and%20not%20deeply%20nested.

---

Bad joke... @eternoendless define the inverse thing : #20324 since the 1.7.7.0.

---

We can change the editorconfig file :sweat_smile:

---

We also say it in our documentation: https://devdocs.prestashop.com/1.7/themes/getting-started/guidelines/#general

> 2 spaces for all other file types.